client/my-sites/comments/comment-navigation/index.jsx

```diff
@@ -60,6 +61,15 @@ export class CommentNavigation extends Component {
 
 	shouldComponentUpdate = nextProps => ! isEqual( this.props, nextProps );
 
+	componentDidUpdate = prevProps => {
+		const { commentsListQuery, hasPendingBulkAction, refreshPage } = this.props;
+		if ( ! hasPendingBulkAction && prevProps.hasPendingBulkAction ) {
+			if ( commentsListQuery ) {
```

Comment:
I guess we might simplify this into
`if ( commentsListQuery && ! hasPendingBulkAction && prevProps.hasPendingBulkAction )`

No reason to even check pending actions if we're not using `commentsListQuery`, and vice versa.